src/clickhouse_migrations/cmd.py

```diff
@@ -28,6 +28,16 @@ def log_level(value: str) -> str:
     raise ValueError
 
 
+def cluster_name(value: str) -> str:
```

Comment:
The best way is duplicate validation which was made by clickhouse cluster. But I think it's not a good solution.
If the user passes an invalid cluster name - let clickhouse through an error about it.
So, I would suggest leaving any issues with the cluster name between the user and clickhouse server.
Its approach makes this function redundant.